scripts/mkcloud

```diff
@@ -540,24 +540,84 @@ function setupadmin()
     fi
 }
 
+function shutdown_node() {
+    virsh shutdown $1
+}
```

Comment:
Well ok, fine.
( sidenote: I expect that we will no longer have this part in bash code when we add other virtualization approaches )